### PR TITLE
Add <FluentHorizontalScroll>

### PR DESCRIPTION
**Pull request by vnbaaij**
# Pull Request

## 📖 Description
This PR adds a `<FluentHorizontalScroll>` component (and associated files) to wrap the `<fluent-horizontal-scroll>` web component

## 👩‍💻 Reviewer Notes
Besides the component itself (`FluentHorizontalScroll.razor`), pages have been added and modified to test the component. See the 'files changed' in this PR.

## 📑 Test Plan
Visist the `FluentHorizontalScroll.razor` page to see several variations on using the toolbar.

### General
- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-blazor/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific
- [x] I have added a new component
- [ ] I have modified an existing component

## ⏭ Next Steps
Review and determine if functionality is sufficient/is missing

### CODE REVIEW

**Comment by EisenbergEffect** [comment]
@vnbaaij Can you help resolve the conflicts on this one?

File: src/Microsoft.Fast.Components.FluentUI/Components/FluentHorizontalScroll.razor
Comment (EisenbergEffect) [inline src/Microsoft.Fast.Components.FluentUI/Components/FluentHorizontalScroll.razor:10]: Should we add some documentation here that indicates what the options are?

**Review by EisenbergEffect — APPROVED** [review]

**Comment by vnbaaij** [comment]
> @vnbaaij Can you help resolve the conflicts on this one?

Sure, if you give me write access... :)

File: src/Microsoft.Fast.Components.FluentUI/Components/FluentHorizontalScroll.razor
Comment (vnbaaij) [reply src/Microsoft.Fast.Components.FluentUI/Components/FluentHorizontalScroll.razor:10]: Or maybe introduce an enum for `Easing`?

Comment (vnbaaij) [reply src/Microsoft.Fast.Components.FluentUI/Components/FluentHorizontalScroll.razor:10]: And perhaps it is better to make `Speed` not nullable and give it a default value (of 600?).

**Comment by EisenbergEffect** [comment]
@vnbaaij Oh! I didn't realize you couldn't access the feature to resolve. I can handle that. Why don't you go ahead and make the changes for Speed and Easing and then after that I'll resolve and merge?

**Comment by vnbaaij** [comment]
I resolved it already with the latest commit. Will work on Speed and Easing next week (couple of days afk)

**Comment by vnbaaij** [comment]
Can't resolve directly from within GH

**Comment by EisenbergEffect** [comment]
Got it. Thanks @vnbaaij !

**Review by nicholasrice — APPROVED** [review]

**Pull request merged**
